watertap/unit_models/electrodialysis_1D.py

```diff
@@ -112,6 +131,62 @@ class Electrodialysis1DData(UnitModelBlockData):
     **False** - exclude pressure change terms.}""",
         ),
     )
+    CONFIG.declare(
+        "pressure_drop_method",
+        ConfigValue(
+            default=PressureDropMethod.none,
+            domain=In(PressureDropMethod),
+            description="Method to calculate the frictional pressure drop in electrodialysis channels",
+            doc="""
+     **default** - ``PressureDropMethod.none``
+
+       .. csv-table::
+           :header: "Configuration Options", "Description"
+
+           "``PressureDropMethod.none``", "The frictional pressure drop is neglected." 
+           "``PressureDropMethod.experimental``", "The pressure drop is calculated by an experimental data as pressure drop per unit lenght."
+           "``PressureDropMethod.Darcy_Weisbach``", "The pressure drop is calculated by the Darcy-Weisbach equation."
+       """,
+        ),
+    )
+    CONFIG.declare(
+        "friction_factor_method",
+        ConfigValue(
+            default=FrictionFactorMethod.defined,
+            domain=In(FrictionFactorMethod),
+            description="Method to calculate the Darcy's friction factor",
+            doc="""
+     **default** - ``FrictionFactorMethod.defined``
+
+       .. csv-table::
+           :header: "Configuration Options", "Description"
+
+           "``FrictionFactorMethod.defined``", "Friction factor is defined and fixed by users" 
+           "``FrictionFactorMethod.Ponzio``", "Frction factor evaluated based on Pazio's work"
+           "``FrictionFactorMethod.Gurreri``", "Frction factor evaluated based on Gurreri's work"
+           "``FrictionFactorMethod.Kuroda``", "Frction factor evaluated based on Kuroda's work"
```

Comment:
"i" missing in "Frction factor"

Reply:
> I have a few comments, questions, and recommended changes.

Thanks Adam!  I just replied to some of your comments.  I will make commits to address the rest.